### ISSUE TITLE
Unilateral sensorineural hearing impairment

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10766]
**Preferred term label:**

**Synonyms**
Unilateral nerve deafness - Unilateral sensorineural deafness - unilateral sensorineural hearing loss - Hearing loss, sensorineural, unilateral 

**Definition (free text, please give PubMed ID)**
PMID: 36630865 Congenital unilateral sensorineural hearing loss (cuSNHL) carries potentially significant social, educational, and developmental consequences. Early diagnosis enables investigation, and consideration of options for management and early intervention, helping to mitigate the effects of hearing loss. Cochlear nerve dysplasia (CND) is a prominent cause of cuSNHL and may affect candidacy for cochlear implantation. 
Cochlear nerve dysplasia is a common cause of unilateral sensorineural hearing loss.

PMID: 16700954 Pure tone audiometry revealed an approximately 55 dB sensorineural hearing loss, with slightly sloppy curving at high frequencies on the left side; the right side audiogram was normal.
The case described presented with airway obstruction and unilateral sensorineural deafness

**Parent term (use hpo.jax.org/app)**
Sensorineural hearing impairment HP:0000407

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cochlear nerve deficiency ORPHA:502318
Hemifacial hyperplasia ORPHA:141145

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 

added as 
HP_0025797
Unilateral sensorineural hearing impairment
